examples/pytorch/vision/Conference_Room_M4/utils/augmentations.py

```diff
@@ -0,0 +1,900 @@
+#-*- coding:utf-8 -*-
```

Comment:
Do we own this file? If so, where is the license header? If not, what is the license for the borrowed file?

This seems different from the augmentation file in Face_Detection? Are the differences to augment SCUTHEAD as opposed to WIDERFACE dataset? If so, can we split this into two files, one to augment each dataset?